llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -995,26 +995,19 @@ static bool isGlobalVarSummary(const ModuleSummaryIndex &Index,
   return false;
 }
 
-// Return the number of global summaries and record the number of function
-// summaries as output parameter.
-static unsigned numGlobalVarSummaries(const ModuleSummaryIndex &Index,
-                                      FunctionImporter::ExportSetTy &ExportSet,
-                                      unsigned &DefinedFS) {
+// Return the number of global summaries in ExportSet.
```

Comment:
"global variable summaries" (to distinguish from global value summaries).

Reply:
done.